third_party/tutorial.proto

```diff
@@ -46,10 +46,10 @@ message BlockContent {
     InfoBox info_box = 3;
     CodeBlock code_block = 4;
     Table table = 6;
-
-    reserved 5; 
-    reserved "horizontal_rule";
   }
+
+  reserved 5; 
+  reserved "horizontal_rule";
```

Comment:
formatting issue: this was submitted before the build system was in place and error checking this file's format